utils/helpers.js

```diff
@@ -0,0 +1,283 @@
+require('dotenv').config();
+
+function delay() {
+    return new Promise(resolve => setTimeout(resolve, process.env.RETRY_DELAY || 2000));
+}
+
+async function getBalance(erc20Contract, tokenAddress, signersAddress) {
```

Comment:
nit: Please use the ES6 function format instead in this file. We should standardize it across but since this is new we can control this
```suggestion
constant getBalance =  async (erc20Contract, tokenAddress, signersAddress) => {
```

Reply:
Done.

Reply:
Done.